shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ContributionResult.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.shortcircuit;
+
+/**
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+public class ContributionResult {
+    private final String id;
```

Comment:
Maybe this class could also have the id of the bus as an attribute

Reply:
I think it will be a duplicate of the one in fault result, so I would not be in favor of this.

Reply:
You have to be more explicit or to increase Javadoc. Here, you don't need the id of the bus but the id of the connectable (load, generator, line, etc.) that has its part in the total amount of three phase current.